cmd/talosctl/cmd/mgmt/gen/config.go

```diff
@@ -280,9 +279,14 @@ func validateFlags() error {
 	return err
 }
 
+// nolint:gocyclo
 func writeConfigBundle(configBundle *bundle.ConfigBundle, outputPaths configOutputPaths, commentsFlags encoder.CommentsFlags) error {
 	outputTypesSet := slices.ToSet(genConfigCmdFlags.outputTypes)
 
+	if err := validateFilesExists([]string{outputPaths.controlPlane, outputPaths.worker, outputPaths.talosconfig}); err != nil {
```

Comment:
I think this might be wrong it `outputPaths` is not complete/stdout. 

I would better keep the check in `writeToDestination`, or even use `writeToDestination` everywhere in `gen` function?